_data/docsarchive/nginx-docs.conf

```diff
@@ -0,0 +1,27 @@
+# prevent buffering to disk
+proxy_buffering off;
+
+# prevent writing large requests to disk
+# values should be equal
+client_body_buffer_size 5m;
+client_max_body_size    5m;
+
+server {
+  listen       80;
+  server_name  docs.docker.com;
+  resolver     127.0.0.11 ipv6=off valid=30s;
+  error_page   404 /404.html;
```

Comment:
Yes, I noticed it in your PR 👍 

I'm not sure we're going ahead with this PR, if the current approach on master works out well enough, but I can update this one to keep it in a mergeable state